packages/fxa-settings/src/lib/oauth/hooks.tsx

```diff
@@ -180,16 +180,22 @@ export function tryAgainError() {
  */
 export function useFinishOAuthFlowHandler(
   authClient: AuthClient,
-  integration: Pick<Integration, 'clientInfo' | 'type' | 'wantsKeys' | 'data'>
+  // TODO: Make this type safe by requiring an OAuthIntegration be provided.
```

Comment:
Is this a follow-up todo?

Reply:
I'm removing this. After looking at how this gets used, this approach seems good enough for now.